recipe/meta.yaml

```diff
@@ -34,6 +34,7 @@ requirements:
     - pygments >=2.7
     - python >=3.8
     - sphinx >={{ min_sphinx }}
+    - typing-extensions
```

Comment:
For reference: https://github.com/pydata/pydata-sphinx-theme/blob/42b297638df611bbf98aea18dcf3044bbb934e8c/pyproject.toml#L30